app/scripts/modules/core/src/presentation/index.ts

```diff
@@ -7,6 +7,7 @@ export * from './hooks';
 export * from './HoverablePopover';
 export * from './LabelComponent';
 export * from './labeledValues';
+export * from './linkWithClipboard.component';
```

Comment:
Should we make the react component available as well?

Reply:
Good call, I will add that for future use!